.github/workflows/build-rust-ci-image.yml

```
@@ -4,13 +4,13 @@ on:
   workflow_dispatch: {}
 
 # Create a GitHub-hosted larger image-generation runner with this label before
-# running this workflow. After it succeeds, configure linux-extra-beefy to use
-# the generated `macro-rust-ci` custom image.
+# running this workflow. Keep its CPU/RAM/disk size at or below the smallest
+# runner size that should be allowed to use the generated custom image.
 jobs:
   build-rust-ci-image:
-    runs-on: linux-extra-beefy-image-generation
+    runs-on: linux-rust-ci-image-generation-small
```

actionlint (1.7.12)

[error] label "linux-rust-ci-image-generation-small" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)